build.gradle

```diff
@@ -68,7 +68,7 @@ dependencies {
 
     compile 'org.thymeleaf:thymeleaf-spring4:3.0.0.RELEASE'
     compile 'org.thymeleaf:thymeleaf:3.0.0.RELEASE'
-    compile 'org.thymeleaf.extras:thymeleaf-extras-springsecurity4:3.0.0.RELEASE'
+    compile 'org.thymeleaf.extras:thymeleaf-extras-springsecurity5:3.0.4.RELEASE'
```

Comment:
[Spring Boot 1.5.18.RELEASE gebruikt Spring Security 4.2.10.RELEASE](https://docs.spring.io/spring-boot/docs/1.5.18.RELEASE/reference/html/appendix-dependency-versions.html), dus kun je Thymeleaf niet zomaar upgraden naar een versie voor Spring Security 5.

Reply:
Ik heb het weer gerevert. De feature werkt nog wel. Ik heb blijkbaar een feature gebruikt die al in 4 zat.